benchbuild/utils/log.py

```diff
@@ -1,5 +1,7 @@
 import logging
 
+import coloredlogs
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'coloredlogs'](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5028678)